espflash/src/image_format/esp8266.rs

```diff
@@ -87,9 +88,16 @@ impl<'a> Esp8266Format<'a> {
             data: Cow::Owned(common_data),
         };
 
+        let app_size = irom_data
+            .clone()
+            .map(|d| d.data.len() as u32)
+            .unwrap_or_default()
+            + flash_segment.data.len() as u32;
```

Comment:
@bjoernQ @MabezDev Do either of you know if this is correct for the ESP8266? I wasn't sure if the IROM data should be included in the total "application" size or not.

Reply:
I believe so, IROM being instruction ROM, aka flash